java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1011,4 +1011,22 @@ public void zcard(BaseClient client) {
                 assertThrows(ExecutionException.class, () -> client.zcard("foo").get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void pfadd(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        assertEquals(1, client.pfadd(key1, new String[] {"one", "two"}).get());
+        assertEquals(0, client.pfadd(key1, new String[] {"two"}).get());
+        assertEquals(0, client.pfadd(key1, new String[0]).get());
+        assertEquals(1, client.pfadd(key2, new String[0]).get());
```

Comment:
You could do this operation with key1 prior to adding "one" and "two".  Then you wouldn't need to use multiple keys.